modules/ROOT/pages/_partials/amql.adoc

```diff
@@ -0,0 +1,233 @@
+// tag::amqlIntro[]
```

Comment:
This file (modules/ROOT/pages/_partials/amql.adoc) is included in the new am-apis.adoc